docs/release-notes/july-2023.md

```diff
@@ -0,0 +1,230 @@
+- Rust: Fixed a bug where standalone metavariable patterns were not matching as expected. ([#8206](https://github.com/returntocorp/semgrep/pull/8206)) 
+- Rust: Macro calls which involve dereferencing and reference operators (such as `foo!(&x)` and `foo!(*x)`) now properly transmit taint.
+- Fixed Python Semgrep pattern parsing to also parse match statements, by chaining in the Python tree-sitter parser, and adding metavariable support to the Python tree-sitter parser.
+- Aliengrep mode: Fix whitespace bug preventing correct matching of parentheses. ([#7990](https://github.com/returntocorp/semgrep/pull/7990))
+- Fixed stack overflow caused by symbolic propagation.
+
+
+### Removed
+
+- **Dart** has been removed from experimental support.
+
+## Semgrep Cloud Platform
+
+### Added
+
+- Jira integration is now in **private beta** for existing customers.
+    - You can **create Jira tickets** from Semgrep Cloud Platform by following steps in the [Jira integration documentation](/semgrep-cloud-platform/jira/).
+    - To enable this feature:
+         1. Fill out the following form: [Request access to the Semgrep Jira integration private beta](https://get.semgrep.dev/Jira-private-beta.html).
+        2. Contact your Technical Account Manager or your Account Executive and let them know you'd like to try Jira integration out.
+- Usage limits are now in effect as of July 31, 2023. See the [Usage limits FAQ](/usage-limits) to learn more.
+- Various bugfixes and improvements.
+
+## Semgrep Code
+
+### Added
+
+- Added **open findings** column to the **Policies** page. This column displays the count of open findings for each rule in your Policies. With this column, you can quickly see which rules are producing the most or least amount of findings.
+- Added **fix rate** column to the **Policies** page. This column displays a percentage of resolved fixes associated with the rule, or  "-" if there are no findings for that rule.
+- Added **Label** as its own column in the **Policies** page.
+- Added icons for **Severity** and **Source** columns in the **Policies** page. The following icons can be seen in the **Source** columns:
+    - <i class="fa-solid fa-pen-nib"></i> Custom rules
+    - <i class="fa-solid fa-diagram-project"></i> Community rules
+    - <i class="fa-solid fa-crown"></i> Semgrep Pro rules
+
+![Icons for Sources and Severities](/img/july2023-newicons.png)
+
+### Changed
+
+- **Policies** page: Rules are now sorted by mode, then by custom rules, then by Semgrep Pro rules, then by Community rules.
+- Targets in a `.yarn/` folder or directory are now ignored by the default `.semgrepignore` patterns.
+
+### Fixed
+
+- Fixed an issue with the **Findings** page > **Triage** button in which the **Ignore** button was previously disabled when the **Note** or **Comment** textbox was empty. It has been fixed to let users Ignore the finding without filling the text box.
```

Comment:
```suggestion
- Fixed an issue with the **Findings** page > **Triage** button in which the **Ignore** button was previously disabled when the **Note** or **Comment** textbox was empty. It has been fixed to let users ignore the finding without filling the text box.
```

Reply:
Ah I should have bolded that Ignore so it's clear it's the text on the button. I'll bold it instead.